### ISSUE TITLE
safety check command returns python warnings

### ISSUE THREAD

**Issue by daurrutia** [pyupio/safety#235]
* safety version: safety 1.8.5
* Python version: Python 2.7.12
* Operating System: Ubuntu 16.04.6 LTS

### Description

Executed `safety check --bare` to return a list of vulnerable pip packages (or nothing).

Output contained python warnings in stderr.

Anyone encounter before?

### What I Did

```
root@ip:/home/ubuntu# safety check --bare
/usr/local/lib/python2.7/dist-packages/packaging/requirements.py:57: UserWarning: warn_ungrouped_named_tokens_in_collection: setting results name 'specifier' on And expression collides with '_original_end' on contained expression
  VERSION_SPEC = originalTextFor(_VERSION_SPEC)("specifier")
/usr/local/lib/python2.7/dist-packages/packaging/requirements.py:60: UserWarning: warn_ungrouped_named_tokens_in_collection: setting results name 'marker' on And expression collides with '_original_end' on contained expression
  MARKER_EXPR = originalTextFor(MARKER_EXPR())("marker")
```

**Comment by leebrian** [comment]
Also seeing this on Python 3.7.4 running in the "python:latest" dockerhub instance.

**Comment by daurrutia** [comment]
resolved by pypa/packaging#170
